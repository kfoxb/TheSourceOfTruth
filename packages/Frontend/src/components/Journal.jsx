import React from 'react';
import EditorViewContainer from '../containers/EditorViewContainer';

export default function Journal() {
  return (<EditorViewContainer />);
}
