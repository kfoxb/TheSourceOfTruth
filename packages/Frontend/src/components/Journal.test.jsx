/* eslint-env jest */
import React from 'react';
import { shallow } from 'enzyme';
import Journal from './Journal';

describe('Journal', () => {
  it('should render', () => {
    expect(shallow(<Journal />)).toMatchSnapshot();
  });
});
