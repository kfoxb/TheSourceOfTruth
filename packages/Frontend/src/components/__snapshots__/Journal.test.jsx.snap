// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Journal should render 1`] = `<EditorViewContainer />`;
